Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,26 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox 
+    {
+        public ValueTuple<int, int> Location { get; }
+        public Person Owner { get; }
+        public Size Size { get; }
+
+
+
+        public Mailbox(ValueTuple<int,int> location, Person owner, Size size) 
```

Comment:
ValueTuple syntax can be simplified
```suggestion
        public Mailbox((int, int) location, Person owner, Size size) 
```